dashboard/client/src/pages/job/JobDetail.tsx

```diff
@@ -238,40 +190,12 @@ export const JobDetailChartsPage = () => {
       <Grid container>
         <Grid item xs={4}>
           <TitleCard title="">
-            <Box
-              mb={2}
-              display="flex"
-              flexDirection="column"
-              height="300px"
-              style={{
-                overflow: "hidden",
-                overflowY: "scroll",
-              }}
-              sx={{ borderRadius: "16px" }}
-            >
-              {cluster_status?.data
-                ? formatNodeStatus(cluster_status?.data.clusterStatus)
-                : "No cluster status."}
-            </Box>
+            <NodeStatusCard cluster_status={cluster_status} />
```

Comment:
I this you code is out of date. I made some changes in latest master. YOu might run into merge conflicts here.